examples/teaching-vm_openrefine/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Integrated example for the Ten Simple Rules for Dockerfiles project, 
+# see https://osf.io/fsd7t/
+# This Dockerfile is a rewrite based on
+# - https://github.com/innovationOUtside/tm351vm/blob/master/build/minimal/Dockerfile
+# - https://github.com/innovationOUtside/tm351vm/blob/master/build/openrefine/Dockerfile
+FROM ubuntu:16.04
+
+RUN apt-get update && \
+  apt-get upgrade -y && \
+  apt-get install -y \
+    # required for installation of OpenRefine    
+    wget \
+    ant \
+    unzip \
+    openjdk-8-jre-headless && \
+  apt-get purge && \
+  apt-get clean -y && \
+  rm -rf /var/lib/apt/lists/*
+
+# Download and unpack OpenRefine ((https://openrefine.org/) to /opt/openrefine-x.y, then delete the downloaded archive
+ENV OPENREFINE_VERSION="3.2"
+# other versions: 2.8, 3.0-beta
+ENV OPENREFINEGZ="openrefine-linux-${OPENREFINE_VERSION}.tar.gz" \
+    OPENREFINESRC="https://github.com/OpenRefine/OpenRefine/releases/download/${OPENREFINE_VERSION}/openrefine-linux-${OPENREFINE_VERSION}.tar.gz"
+RUN wget -q --no-check-certificate  -P /root $OPENREFINESRC && \
+  tar -xzf /root/$OPENREFINEGZ -C /opt && \
+  rm /root/$OPENREFINEGZ
+
+# Create working directly and expose it as a volume
+WORKDIR /openrefine_projects
+VOLUME /openrefine_projects
+
+LABEL maintainer="tony.hirst@gmail.com" \
+  org.opencontainers.image.authors="Tony Hirst" \
+  org.opencontainers.image.url="https://github.com/innovationOUtside/tm351vm"
```

Comment:
Should both OU be capitalized?

Reply:
I copied that from the existing README...

Reply:
Just checked - not a typo! https://github.com/innovationOUtside I sense you are angry with me, I don't mean to make you upset, I'm trying to give a thorough review.

Reply:
I know - thanks! :pray: